libs/expo/betterangels/src/lib/screens/Clients/index.tsx

```diff
@@ -35,7 +35,7 @@ export default function Clients({ Logo }: { Logo: ElementType }) {
   const [createNote] = useCreateNoteMutation();
   const [offset, setOffset] = useState<number>(0);
   const [hasMore, setHasMore] = useState<boolean>(true);
-  const { data, loading } = useClientProfilesQuery({
+  const { data, loading } = useClientProfilesPaginatedQuery({
     variables: {
       pagination: { limit: paginationLimit + 1, offset },
```

Comment:
> We don't need to do +1 anymore given we now know how many results there are. We originally used the over-fetching approach since we didn't know the total count.
https://github.com/BetterAngelsLA/monorepo/pull/822#discussion_r1915419752